app/src/repositories/user/userRepository.js

```diff
@@ -47,23 +47,66 @@ class UserRepository {
 
   async updateNickname(userId, newNickname) {
     const query = `
-    UPDATE users
-    SET nickname = ?
-    WHERE id = ?;
-  `;
+      UPDATE users
+      SET nickname = ?
+      WHERE id = ?;
+    `;
     const [result] = await pool.query(query, [newNickname, userId]);
     return result.affectedRows === 1;
   }
 
   async updatePassword(userId, hashedPassword) {
     const query = `
-    UPDATE users
-    SET password = ?
-    WHERE id = ?;
-  `;
+      UPDATE users
+      SET password = ?
+      WHERE id = ?;
+    `;
     const [result] = await pool.query(query, [hashedPassword, userId]);
     return result.affectedRows === 1;
   }
+
+  async findFavoritesByUserId(userId, sort) {
+    const allowedSorts = {
+      recent: "wf.created_at DESC",
+      updated: "w.updated_at DESC",
+    };
```

Comment:
1. 어플리케이션 생명주기간 변하지 않는 상수는 UPPER_SNAKE_CASE로 선언되면 좋을거같아요
2. recent가 생성일자 기준 정렬일텐데 id가 auto increment 또는 시간기준 uid인 경우에는 id에 인덱스가 있기 때문에 효율이 더 좋습니다

```suggestion
    const ALLOWED_SORTS = {
      RECENT: "wf.created_at DESC",
      UPDATED "w.updated_at DESC",
    };
```

Reply:
validator를 보니 query에서 받은 값을 그대로 사용해서 recent, updated를 사용하신거같습니다.

레이어마다 필요한 변수명을 요구하고 그걸 받는것이 좋지만 당장에 그런 작업이 까다롭다면 굳이 변경하지 않아도 좋을거같아요

Reply:
항상 리뷰 덕분에 많이 배웁니다. 리뷰 남겨주셔서 감사해요!
상수 처리, id(auto increment)로 정렬하면 더 효율적이라는 점, 그리고 query 값을 레이어별로 변환해서 사용한다는 점까지 참고하고 적용해보겠습니다.